### ISSUE TITLE
Correct FRC acronym branding

### ISSUE THREAD

**Issue by Kevin-OConnor** [wpilibsuite/frc-docs#250]
The registered trademark symbol should be added to the first use of the FRC acronym in body text on any page where it appears.

This can be accomplished by using

`..include:: <isonum.txt>` at the top of the document and the using `FRC\ |reg|` at the first usage of the acronym

**Comment by sciencewhiz** [comment]
There should be a space between .. and include: `.. include:: <isonum.txt>`

**Comment by Daltz333** [comment]
Are there areas where this is still needed? Can someone do a grep?

**Comment by AustinShalit** [comment]
(Checking acronym formatting would be a great candidate for a check in CI. It could also check all of the other acronyms that we use)

**Comment by Daltz333** [comment]
I think that would be best as part of a formatter. #228

**Comment by jasondaming** [comment]
So what all does this apply to.  I see:
FIRST
FRC
any others? (I would imagine it could (maybe should) apply to: Microsoft, NI, Axis, CTRE, Vex, etc.

**Comment by jasondaming** [comment]
what about the FRC Game Manual at the top of every page?  I assume that needs it too?

**Comment by jasondaming** [comment]
does frc-docs need to have it also?

**Comment by Daltz333** [comment]
frc-docs is creative commons

**Comment by jasondaming** [comment]
Note that the ..include:: <isonum.txt> mentioned above will not work.  There must be a space between .. and include

**Comment by ofekashery** [comment]
``FRC®`` is the former name and it should appear as little as possible in official use.
According to the _FIRST_ Branding, it should be written out as <em>FIRST</em><sup>®</sup> Robotics Competition, not FRC.

**Comment by Daltz333** [comment]
That's not really feasible for us. @Kevin-OConnor can you confirm

**Comment by Kevin-OConnor** [comment]
Yes, we are trying to minimize use of the acronym, why is it not feasible?

**Comment by Daltz333** [comment]
1. We probably don't want translators to translate it
2. Seems to add a lot of extra verbosity. Can we use FRC in titles? Or should we just avoid using FRC/FIRST Robotics Competition at all in titles.

**Comment by jasondaming** [comment]
I currently count 1514 uses of FRC in the docs.  Not to mention all of the Titles and such it would legthen immensely

**Comment by Daltz333** [comment]
We *can* do it, but it'd be a long term thing to work toward. I think an appropriate short-term solution would be added a notice to the style guide to avoid FRC/FIRST Robotics Competition in titles, and use FIRST Robotics Competition as we go, updating references in documents that are updated. Or we could use the spell-checker to warn on updated documentation to update.

**Comment by Kevin-OConnor** [comment]
I agree, I am not suggesting we just Find/Replace it as that will add a ton of length. I think it is a longer term trying to rephrase things to say it less if they're being touched anyway kind of thing.

**Comment by Daltz333** [comment]
I think the following will close this issue:
1. Add a notice to the style guide to use FIRST Robotics Competition and not FRC. Do not use either in titles.
2. Add someway (Github actions check) to flag certain spellings of words (FRC) and to use alternative. Perhaps a tool like wpiformat. A simple python script in ``scripts`` would be fine.

**Comment by Kevin-OConnor** [comment]
Yep, that sounds reasonable
